readinglist/resource.py

```diff
@@ -129,26 +129,44 @@ def _extract_filters(self, queryparams):
         filters = []
 
         for param, value in queryparams.items():
+            param = param.strip()
             value = native_value(value)
 
-            m = re.match(r'^(min|max|not)_(\w+)$', param.strip())
+            if param == '_sort':
+                continue
+
+            if param == '_since':
+                if not isinstance(value, six.integer_types):
+                    error_details = {
+                        'name': param,
+                        'location': 'querystring',
+                        'description': 'Invalid value for _since'
+                    }
+                    self.request.errors.add(**error_details)
+                    return
+
+                filters.append(
+                    (self.modified_field, value, COMPARISON.MIN)
```

Comment:
Should we use [names tuples](https://docs.python.org/2/library/collections.html#collections.namedtuple) instead of triples?